packages/crayons-core/src/components/form/readme.md

```diff
@@ -3259,6 +3259,7 @@ graph TD;
   fw-form-control --> fw-select
```

Comment:
Please add the code samples in the documentation for this type

Reply:
Done.